kernel/lib/printf.cpp

```diff
@@ -341,17 +335,20 @@ int px_ksprintf(char* buf, const char* fmt, ...)
 }
 /*****************************************************************************
 *****************************************************************************/
-int vprintf_help(unsigned c, void** ptr)
+static int vprintf_help(unsigned c, void** ptr)
```

Comment:
Do we want to start prepending `_` to private functions?

Reply:
We can. It was an experiment I started in the task/scheduling code that we can decide whether or not to carry over to everything else. Generally marking something as `static` is good enough, but I didn't want to get confused between the public and private versions of certain scheduling functions.

Reply:
Sounds good. I'll make an issue for it.